README.md

```diff
@@ -0,0 +1,32 @@
+#  Tkey ios example application document
+
+After building complete, you can login via your google account.
+Clicking this button will make account for you, using customAuth sdk.
```

Comment:
No, existing accounts can be used as well, caution should be advised when doing this though.